hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/uchitsa/hw01_hello_otus
```

Рекомендую указать полный путь до модуля
```suggestion
module github.com/uchitsa/go-otus-homeworks/hw01_hello_otus
```